README.md

```diff
@@ -115,8 +115,8 @@ Resource are the objects that are going to be used for steps in the scenario. Th
   * [MySQL](https://github.com/tomatool/tomato/blob/master/docs/resources/database-sql.md)
   * [PostgreSQL](https://github.com/tomatool/tomato/blob/master/docs/resources/database-sql.md)
 * [HTTP Client](https://github.com/tomatool/tomato/blob/master/docs/resources/http-client.md) - to send HTTP Request
-* [HTTP Server](https://github.com/tomatool/tomato/blob/master/docs/resources/http-server.md) - to create fake external service dependency that serve HTTP API
 * [Message Queue](https://github.com/tomatool/tomato/blob/master/docs/resources/message-queue.md) - to manage (publish/consume/check) message queue
   * [RabbitMQ](https://github.com/tomatool/tomato/blob/master/docs/resources/message-queue.md)
   * [NSQ](https://github.com/tomatool/tomato/blob/master/docs/resources/message-queue.md)
+  * [Wiremock](https://github.com/tomatool/tomato/blob/master/docs/resources/wiremock.md)
```

Comment:
wiremock is not part of `Message Queue` resource, I think it should be on `HTTP Server` resource